bundle.js

```diff
@@ -19,38 +20,69 @@ function deleteDirectory(directory) {
     }
 }
 
-function replaceFileContents(string, fileDir) {
-    const pattern = /%%([^%]+)%%|\/\/\s*include\s*"([^"]+)"/g;
```

Comment:
I have not seen the second variant of the pattern (`include "path"`) used anywhere, so I have removed it, for simplicity.